### PR TITLE
WIP docs improvements (village tracker)

### PR DESCRIPTION
**Pull request by mixmix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,69 +6,89 @@ and pull-streams.
...
 that is probably a bug. The above methods will act like the empty file is already in the store.
 
+### blobs.meta
+
+???
+
+### blobs.resolve
+
+???
+
```
Comment (mixmix) [inline README.md:91]: these are straight up missing !

**Comment by dominictarr** [comment]
clarification: are you gonna work on this more or should I just merge it now?

**Comment by mixmix** [comment]
Let's hold till the end of the village-tracker week and then merge where I get to !

**Comment by dominictarr** [comment]
merged into 1.13.4! thanks

**Pull request merged**